<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>284</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <layout class="QVBoxLayout" name="none">
      <item>
       <widget class="QTextBrowser" name="chatHistory"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="none_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Message</string>
          </property>
          <property name="buddy">
           <cstring>messageText</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="messageText"/>
        </item>
        <item>
         <widget class="QPushButton" name="sendBtn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Send</string>
          </property>
          <property name="autoDefault">
           <bool>true</bool>
          </property>
          <property name="default">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>584</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>messageText</tabstop>
  <tabstop>sendBtn</tabstop>
  <tabstop>chatHistory</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
